js/app.js

```diff
@@ -78,13 +77,34 @@ function render() {
   console.log(totalClicks);
   comparisonValues.splice(0,3);
   if (totalClicks < 1) {
-    ulEl.removeEventListener('click', render);
-    //call stats method
+    ulEl.removeEventListener('click', handleClicks);
+    listCreator();
   }
 }
 
+//Sam method for handling clicks for the pictures.
+function handleClicks(event) {
+  for (var i = 0; i < ImageCreator.allImages.length; i++) {
+    if (event.target.alt === ImageCreator.allImages[i].title) {
+      ImageCreator.allImages[i].clicks += 1;
+    }
+  }
+  render();
+}
+
+//Target the listContainer id to create new li elements that then propagate the data to display.
+function listCreator() {
+  for(var i = 0; i < ImageCreator.allImages.length; i++) {
+    var liEl = document.createElement('li');
+    liEl.textContent = ImageCreator.allImages[i].clicks + ' votes for the ' + ImageCreator.allImages[i].title;
+    ulEl2.appendChild(liEl);
+  }
+}
+
+
+//Functions that happen on page load
 render();
 randomGen();
```

Comment:
Pretty sure that you do not need to call `randomGen` here. The only thing it does is return a random number and you are not using that for anything here.